### PR TITLE
Migrated some tests from ami-val.

### PR DESCRIPTION
**Pull request by narmaku**

### CODE REVIEW

File: test_suite/cloud/test_aws.py [hunk 1]
```diff
@@ -356,7 +395,7 @@ def test_network_ipv6_setup(self, host):
         assert registered_ipv6 in host.interface('eth0', 'inet6').addresses(), \
             f'Expected IPv6 {registered_ipv6} is not being used by eth0 network adapter'
 
-    def test_redhat_cds_hostnames(self, host, instance_data_aws):
+    def test_redhat_cds_hostnames(self, host, instance_data_aws, rhel_only):
```
Comment (narmaku) [inline test_suite/cloud/test_aws.py:398]: Sorry, I was doing some tests here.

```suggestion
    def test_redhat_cds_hostnames(self, host, instance_data_aws):
```

File: test_suite/generic/test_generic.py [hunk 2]
```diff
@@ -173,6 +173,13 @@ def test_release_version(self, host, instance_data):
             f'product version ({product_version}) does not match package release version'
         assert product_version.replace('.', '-') in cloud_image_name, 'product version is not in image name'
 
+    def test_timezone_is_utc(self, host):
+        """
+        Check that the default timezone is set to UTC.
+        BugZilla 1187669
+        """
+        assert 'UTC' in host.check_output('date'), 'Unexpected timezone. Expected to bne UTC'
```
Comment (narmaku) [inline test_suite/generic/test_generic.py:181]:
```suggestion
        assert 'UTC' in host.check_output('date'), 'Unexpected timezone. Expected to be UTC'
```

File: test_suite/cloud/test_aws.py [hunk 3]
```diff
@@ -302,6 +302,45 @@ def test_aws_instance_identity(self, host, instance_data, instance_data_aws):
...
+
+    def test_udev_kernel(self, host):
+        """
+        /etc/udev/rules.d/80-net-name-slot.rules link to /dev/null
+        ks ref:
+        For cloud images, 'eth0' _is_ the predictable device name, since
+        we don't want to be tied to specific virtual (!) hardware
+        rm -f /etc/udev/rules.d/70*
+        ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules
+        """
```
Comment (jabia99) [inline test_suite/cloud/test_aws.py:331]: I don't think this description helps. Maybe we could ask VirtQE to write a more clear onw

**Review by jabia99 — APPROVED** [review]
LGTM

**Pull request merged**